src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -58,11 +64,28 @@ const props = defineProps({
     type: String,
     default: 'Clear filters',
   },
+  areFiltersHidden: {
+    type: Boolean,
+    default: () => undefined,
+  },
+  hiddenFiltersNames: {
+    type: Array,
+    default: () => [],
+  },
 });
 
 const emit = defineEmits(['update:modelValue', 'change']);
+const areFiltersHiddenThreshold = useMediaQuery('(max-width: 768px)');
 
+const alwaysShownFilters = props.filters ? props.filters.filter(filter => (!props.hiddenFiltersNames.includes(filter.name))) : [];
 const hasFilters = computed(() => !!Object.keys(props.modelValue).length);
+const canHideFilters = computed(() => (props.areFiltersHidden || (props.areFiltersHidden === undefined && areFiltersHiddenThreshold.value)));
```

Comment:
I don't think we should have this logic here, let's base hidden state for every filter instead of introducing the hiddenFilterNames prop.

So we have filters prop and can pass idHidden state for every filter.